zerver/lib/export.py

```diff
@@ -352,13 +352,15 @@ def sanity_check_output(data: TableData) -> None:
 
 
 def write_data_to_file(output_file: Path, data: Any) -> None:
+    logging.info("Writing to %s", output_file)
```

Comment:
Maybe just skip the "to" -- "Writing realm.json" or whatever feels pretty readable.

Reply:
I guess.